generators/client/templates/_package.json

```diff
@@ -8,6 +8,7 @@
     "<%= MAIN_SRC_DIR %>bower_components"
   ],
   "devDependencies": {
+    "bower": "1.7.9",
```

Comment:
In the JS world people are moving away from global npm packages and are using devDependencies instead.  http://www.joezimjs.com/javascript/no-more-global-npm-packages/

I really don't understand why you consider bower an unnecessary dependency. 1 bower dependency (that I imagine will probably will be removed soon anyway as more people are moving away from bower and using full npm) will not have a big impact. 

But we don't have to drag this discussion on no longer than it needs to be. 

We can agree to disagree :)